### DISCUSSION TITLE
【Performance analysis】when I monitor the specified endpoints,  it didn't work. No data

### DISCUSSION THREAD

**Discussion by little-hang** [apache/skywalking#8626] [Q&A]
The UI version: 8.7.0
The OAP-server version: 8.9.1
![image](https://user-images.githubusercontent.com/24517540/157001539-1adf7b52-443d-482e-a3e3-c34a62694dfe.png)

I use the 【Performance analysis】to monitor my specified endpoints as follows:
 1. create new task
 
![image](https://user-images.githubusercontent.com/24517540/157001147-2443391a-9eae-4b72-8e91-7a2d139d34eb.png)

 2. curl the specified endpoints's API (more than five times)
 
![image](https://user-images.githubusercontent.com/24517540/157001247-f02040f6-6aa0-4da5-ad93-2222a4f0cc78.png)

 3. touch the view detail in UI 
 
![image](https://user-images.githubusercontent.com/24517540/157001366-9694a0f8-9f04-484d-b0ce-8073cc8ff2f5.png)

 
And then, no data in right UI :
![image](https://user-images.githubusercontent.com/24517540/157001477-e7361d32-7ca3-4406-899f-70c06b685580.png)

**Comment by wu-sheng** [comment]
Only Java and Python agents support profiling. I don't know what happens, but profiling is not 100% sampling, it is caught randomly.
